db/migrate/20160716091234_add_autograded_to_course_assessments.rb

```diff
@@ -0,0 +1,5 @@
+class AddAutomaticallyReleaseGradeToCourseAssessments < ActiveRecord::Migration
```

Comment:
Need to change class name to `AddAutogradedToCourseAssessments` or migration will fail